consume-more-stuff/src/app/component/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,10 @@
+<h2><a routerLink="/">Home</a></h2>
```

Comment:
You don't want to use header tags in this way. They're generally for just bolding text as obvious text headers to content sections. You'll want to use:
```
<ul>
  <li><a href=""></li>
</ul>
```